analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ExecutingSqlQueries_Net46.vb

```diff
@@ -164,5 +151,36 @@ Namespace Tests.Diagnostics
             command.CommandText = String.Format("INSERT INTO Users (name) VALUES (""{0}"")", param) ' Noncompliant
         End Sub
 
+        Public Sub Bar(ByVal connection As SqlConnection, ByVal param As String)
```

Comment:
To follow the style of previous methods. (I personally don't like `Foo` & `Bar` unless it has absolutely no meaning for the test, like an outer class name when the test is related to a method)
```suggestion
        Public Sub ConcatAndStringFormat(ByVal connection As SqlConnection, ByVal param As String)
```